src/utilities/theme/utils.ts

```diff
@@ -225,6 +225,27 @@ function overrides() {
     [toCssCustomPropertySyntax('focusRingContent')]: "''",
     [toCssCustomPropertySyntax('borderRadiusBase')]: rem('4px'),
     [toCssCustomPropertySyntax('borderRadiusWide')]: rem('8px'),
+    [toCssCustomPropertySyntax('borderBox')]: 'border-box',
```

Comment:
I can't think of when we'd ever want to stray away from border-box - we set it globally on `*, *:after, *:before` so why allow configuration here?

Reply:
With the updated banners, `border-box` was being set explicitly so this was to keep it backwards compatible: `box-sizing: var(--p-border-box, inherit);`. Will take another looks because I don’t think this is needed.

Reply:
This is actually an unnecessary style (since we already set it globally). Going to burn it.